### ISSUE TITLE
add option `ignoreRule` to support ignore some specific selector

### ISSUE THREAD

**Issue by lovetingyuan** [kristerkari/css-to-react-native-transform#199]
```css
.foo {
  color: red;
}
.bar {
  font-size: 12px;
}
```

```js
css2rn(csscode, {
  ignoreRule(selector) {
    if (selector === '.foo') { return true }
  }
})
```

result:
```js
{
  "bar": { fontSize: 12 }
}
```

**Comment by kristerkari** [comment]
Thanks! This could be a useful option to add to the library.

**Comment by kristerkari** [comment]
This is now released in 2.1.0:
https://github.com/kristerkari/css-to-react-native-transform/releases/tag/v2.1.0
